perses/rjmc/geometry.py

```diff
@@ -1921,12 +1921,23 @@ def add_atoms(new_atoms, atoms_torsions):
                 The log torsion cchoice probability associated with these added torsions.
 
             """
+            from scipy import special
             logp_torsion_choice = 0.0
             while(len(new_atoms))>0:
                 eligible_atoms = self._atoms_eligible_for_proposal(new_atoms, atoms_with_positions)
+
+                #randomize positions
+                eligible_atoms_in_order = np.random.choice(eligible_atoms, size=len(eligible_atoms), replace=False)
```

Comment:
I guess I could. I know that by drawing without replacement uniformly, the probability of that configuration will be 1/n!, so I just did that.